components/MechPage/MechAgentMetrics.jsx

```diff
@@ -1,24 +1,159 @@
-import { SECTION_BOX_CLASS } from 'common-util/classes';
+import { getMechRqs } from 'common-util/api/dune';
+import { MECH_ECONOMY_DASHBOARD_URL } from 'common-util/constants';
 import SectionWrapper from 'components/Layout/SectionWrapper';
-import { Card } from 'components/ui/card';
+import { ExternalLink } from 'components/ui/typography';
 import Image from 'next/image';
-import { CARD_BG } from 'styles/globals';
-
-export const MechAgentMetrics = () => (
-  <SectionWrapper customClasses={`${SECTION_BOX_CLASS} lg:py-14`}>
-    <div className="max-w-[750px] mx-auto text-center">
-      <Card className={`${CARD_BG} max-h-[250px] p-16`}>
-        <Image
-          alt="Mech metrics"
-          src="/images/metrics.svg"
-          height={48}
-          width={48}
-          className="mb-6 mx-auto"
-        />
-        <p className="text-slate-500">
-          Olas Mech agent economy metrics coming soon.
+import { useMemo } from 'react';
+import useSWR from 'swr';
+
+const fetchMetrics = async () => {
+  const result = await getMechRqs();
+
+  return {
+    totalRqs: result.totalRqs,
+    predictRqs: result.predictRqs,
+    contributeRqs: result.contributeRqs,
+    governatooorRqs: result.governatooorRqs,
+    otherRqs: result.otherRqs,
+  };
+};
+
+const usePersistentSWR = (key, fetcher) =>
```

Comment:
can be made reusable, but minor

Reply:
i'd rather switch to react tanstack like in mono repo